<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>774</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STR CONTROLLER</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>60</y>
      <width>181</width>
      <height>481</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>179</width>
       <height>479</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QComboBox" name="baudRate">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>baudRate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QComboBox" name="dataBits">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>dataBits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QComboBox" name="parity">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>parity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QComboBox" name="stopBits">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>stopBits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QComboBox" name="flowControl">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>FlowControl</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="Com_Uart">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>ROG Fonts</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Com_Uart</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>161</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_SetUart">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_closeUart">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>540</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>179</width>
       <height>189</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btn_Clear">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>60</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="SetSpeed">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Set Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="Set_point">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_setSpeed">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>110</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
       </font>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>ROG Fonts</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Real Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="RealSpeed"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1041</width>
      <height>671</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1039</width>
       <height>669</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>671</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QCustomPlot" name="customPlot" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
